client/controllers.js

```diff
@@ -1,4 +1,43 @@
-var micControllers = angular.module('micControllers', []);
+var micControllers = angular.module('micControllers', ['ribbitBaseRTC', 'ngSanitize']);
+
+// Audience RTC:
+//  - Acquires the user's local media stream right away, upon instantiating
+//  - Connects specifically to the room's presenter.
+// NOTE: The connect method must be passed a room that has a presenter property.
+micControllers.factory('audienceRTC', function (baseRTC) {
```

Comment:
We actually use the code in controllers.js. Originally, I wanted to keep the same clean separation. But, I was getting a provider not defined error for audienceRTC -> audienceRTCProvider so i figured the factory wasn't loading properly (even though I tried different othere's script tags and stuff). Ultimately we should be able to remove it front controllers and use the separate files.